### PR TITLE
Bump OpenTracing to 0.33.0

### PR DESCRIPTION
**Pull request by pavolloffay**
Signed-off-by: Pavol Loffay <ploffay@redhat.com>

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -47,9 +47,9 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 
     <!-- versions -->
-    <version.io.opentracing-opentracing>0.32.0</version.io.opentracing-opentracing>
-    <version.io.opentracing.contrib-opentracing-concurrent>0.3.0</version.io.opentracing.contrib-opentracing-concurrent>
-    <version.io.opentracing.contrib-opentracing-web-servlet-filter>0.3.0</version.io.opentracing.contrib-opentracing-web-servlet-filter>
+    <version.io.opentracing-opentracing>0.33.0</version.io.opentracing-opentracing>
+    <version.io.opentracing.contrib-opentracing-concurrent>0.4.0</version.io.opentracing.contrib-opentracing-concurrent>
```
Comment (pavolloffay) [inline pom.xml]: These just have been released. It might take some time to propagate to central

Comment (kasinskim) [reply pom.xml]: @pavolloffay Any chance for merging/releasing this PR?  It looks like all required dependencies are already in central. Thanks

Comment (pavolloffay) [reply pom.xml]: @kasinskim four curiosity what client are you using?

Comment (kasinskim) [reply pom.xml]: I'm at the very beginning of tracing journey. Currently piloting `jaeger` implementation (for both Java/Node technologies).

BTW. Do you have any plans on providing instrumentation for https://github.com/open-telemetry?

**Comment by tobiasstadler** [comment]
Anyone willing to commit this?

**Comment by pavolloffay** [comment]
This was on hold because the library is used on Smallrye: MicroProfile-Opentracing implementation. 

We will merge it and mantain 0.31.0 in a separate branch.

**Comment by pavolloffay** [comment]
This was on hold because the library is used on Smallrye: MicroProfile-Opentracing implementation. 

We will merge it and mantain 0.31.0 in a separate branch.

**Pull request merged**